src/core/lib/iomgr/call_combiner.h

```diff
@@ -109,4 +110,79 @@ void grpc_call_combiner_set_notify_on_cancel(grpc_call_combiner* call_combiner,
 void grpc_call_combiner_cancel(grpc_call_combiner* call_combiner,
                                grpc_error* error);
 
+namespace grpc_core {
+
+// Helper for running a list of closures in a call combiner.
+//
+// Each callback running in the call combiner will eventually be
```

Comment:
Would it be possible in a future optimization to "know" that the closures all need to be run under this call combiner, and then run them sequentially without having to go through the pattern of; surface yields combiner, then next closure reenters?

that would eliminate the overhead of the combiner re-aquisition